test/database.ts

```diff
@@ -2961,7 +2964,9 @@ describe('Database', () => {
 
       assert.strictEqual(session, SESSION);
       assert.strictEqual(transaction, TRANSACTION);
-      assert.strictEqual(runFn, fakeRunFn);
+      // Given that we've wrapped the transaction runner with observability
+      // tracing, directly comparing values runFn and fakeRunFn.
+      // assert.strictEqual(runFn, fakeRunFn);
```

Comment:
is it possible to add a check on some properties of these functions, instead of removing this check ?

Reply:
Sadly not, unless we are perform a traversal by utility examination but I found that to be overkill and not worth it TBH